docs/eventing/broker/broker-types.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Broker types"
+weight: 01
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - docs/eventing/broker/alternate
+---
+
+The following broker types are available for use with Knative Eventing.
+
+## Multi-tenant channel-based broker
+
+Knative Eventing provides a multi-tenant (MT) channel-based broker implementation that uses channels for event routing.
+
+Before you can use the multi-tenant (MT) channel-based broker implementation, you must install a [channel provider](../channels/channel-types-defaults).
```

Comment:
We no longer call them "channel providers", I think this was old language when we had the bad version of channels. I think we should just call them "channel implementation".